doc/manuals/work_queue/examples/work_queue_abstractions.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+# Example of how to use work_queue abstractions
+import work_queue as wq
+
+def main():
+    # Set up queue
+    q = wq.WorkQueue(port=9223)
```

Comment:
port=9123   as that's the default everywhere else.

Reply:
Oh shoot Sorry I meant to change that but forgot. Thanks